src/components/Content/SideBar.jsx

```diff
@@ -0,0 +1,46 @@
+/* eslint react/no-unused-prop-types: 0 */
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+
+import Links from '../Links/Links';
+
+const linksData = {
+  title: 'Program Documents',
+  links: [
+    {
+      title: 'Degree Requirements',
+      href: 'https://edx.org',
+    },
+    {
+      title: 'Student Record',
+      href: 'https://edx.org',
+    },
+  ],
+};
+
+class SideBar extends Component {
+  state = {};
+
+  render() {
+    return (
+      <React.Fragment>
+        <div className="ml-5">
+          <Links title={linksData.title} links={linksData.links} />
+        </div>
+      </React.Fragment>
+    );
+  }
+}
+
+SideBar.defaultProps = {
+  links: [],
+};
+
+SideBar.propTypes = {
+  links: PropTypes.arrayOf(PropTypes.shape({
```

Comment:
The `Links` component is getting its data from a local variable, so no need for the `propTypes` definition of `links` here. Likewise for `defaultProps`.

Reply:
Removed